File: File Upload for Small Files (up to 50KB)/Upload Small File from Url/JavaScript/UploadSmallFileFromUrl.js
//*******************************************************************************************//
//                                                                                           //
// Download Free Evaluation Version From: https://bytescout.com/download/web-installer       //
//                                                                                           //
// Also available as Web API! Get Your Free API Key: https://app.pdf.co/signup               //
//                                                                                           //
// Copyright © 2017-2020 ByteScout, Inc. All rights reserved.                                //
// https://www.bytescout.com                                                                 //
// https://pdf.co                                                                            //
//                                                                                           //
//*******************************************************************************************//


var request = require('request');

// You can also upload your own file into PDF.co and use it as url. Check "Upload File" samples for code snippets: https://github.com/bytescout/pdf-co-api-samples/tree/master/File%20Upload/    
var options = {
  'method': 'POST',
  'url': 'https://api.pdf.co/v1/file/upload/url',
  'headers': {
    'x-api-key': '{{x-api-key}}'
  },
  formData: {
    'name': 'sample.pdf',
    'url': 'https://bytescout-com.s3.amazonaws.com/files/demo-files/cloud-api/pdf-split/sample.pdf'
  }
};
request(options, function (error, response) {
  if (error) throw new Error(error);
  let data = JSON.parse(response.body);
  console.log(data);
});

File: File Upload for Small Files (up to 50KB)/Upload Small File from Url/Python/UploadSmallFileFromUrl.py
import requests

url = "https://api.pdf.co/v1/file/upload/url"

payload = {'name': 'sample.pdf',
'url': 'https://bytescout-com.s3.amazonaws.com/files/demo-files/cloud-api/pdf-split/sample.pdf'}
files = [

]
headers = {
		'x-api-key': '{{x-api-key}}'
}

response = requests.request("POST", url, headers=headers, data = payload, files = files)

print(response.text.encode('utf8'))
